source/routing/raptor_api.cpp

```diff
@@ -1150,9 +1153,8 @@ void make_isochrone(navitia::PbCreator& pb_creator,
     datetime = tmp_datetime.front();
     worker.init(origin);
     auto departures = get_stop_points(origin, raptor.data, worker);
-
-    if(departures.size() == 0){
-        pb_creator.set_response_type(pbnavitia::NO_ORIGIN_POINT);
+    if (departures == nullptr) {
```

Comment:
you shouldn't check the equality to `nullptr`, only the boolean conversion
`if (departures) {`